NitroxServer/GameLogic/Entities/Spawning/BatchEntitySpawner.cs

```diff
@@ -28,6 +32,14 @@ public class BatchEntitySpawner : IEntitySpawner
         private readonly object emptyBatchesLock = new object();
         private HashSet<NitroxInt3> parsedBatches;
 
+        /// <summary>
+        /// Should not be modified without testing (a good way is "goto wreck13" and look around the wreck)
+        /// 0.1 seemed to be a bit low and 0.3 seemed to be too much
+        /// </summary>
+        private const float FRAGMENT_LUCK_INDICATOR = 0.2f;
+        private const float DEFAULT_LUCK_INDICATOR = 0.1f;
```

Comment:
As discussed in discord this is not a valid option as the "luck indicator" changes with progression and fragment type afair.